.editorconfig

```diff
@@ -0,0 +1,37 @@
+# http://editorconfig.org
```

Comment:
@dhruvkb I have configured the indent_values with the present files of js and python and no such change has occured while formatting the files.